prometheus.yml

```diff
@@ -0,0 +1,18 @@
+global:
+    scrape_interval: 15s
+    evaluation_interval: 15s
+
+scrape_configs:
+    - job_name: 'prometheus'
+      static_configs:
+          - targets: ['localhost:9090']
+    - job_name: 'testunit'
+      static_configs:
+        - targets: ['testunit']
```

Comment:
When running docker-compose up (and looking at the bash files) it seems that testunit and teste2e get destroyed once they run. Should Prometheus be monitoring these?

Reply:
@vaibhavjayaraman I don't think so unless you're adding tests for Athens to spit out metrics?

Reply:
Can you clarify this question for me? Right now I interpret the question to mean that we should include tests that check if the prometheus exporter is working properly by checking if it is spitting out the metrics that we want.

Reply:
If you wish to add tests for prometheus you are welcome to do so, but if not then you do not need to have prometheus monitor the end to end tests